src/Mysqldump.php

```diff
@@ -772,7 +785,7 @@ private function listValues(string $tableName)
         $resultSet = $this->conn->query($stmt);
         $resultSet->setFetchMode(PDO::FETCH_ASSOC);
 
-        $ignore = $this->settings->isEnabled('insert-ignore') ? '  IGNORE' : '';
```

Comment:
@cfuentessalgado look at this - there are 2 spaces. This is something I fought before as it' kinda hard to see. So it needs to be `INSERT  IGNORE` to the tests pass.

Reply:
Well now I remember. Your code works when running the tests on local (docker). And they fail in CI pipeline. Now I need to check how to get it working on both. Which I don't remember 😅